src/screens/WhotoFollow/index.js

```diff
@@ -138,13 +141,14 @@ const WhotoFollow = () => {
     Analytics.logEvent('onb_select_follows_btn_add', {
       onb_whofollow_users_selected: followed
     });
+    AnalyticsEventTracking.eventTrack(BetterSocialEventTracking.ONBOARDING_TOPICS_TOTAL_FOLLOWING, {
+      total: followed?.length
+    });
     const data = {
       users: {
         username: usersState.username,
         human_id: usersState.userId,
         country_code: usersState.countryCode,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new analytics event tracking seems to be correctly implemented. However, it's important to ensure that `followed` is always an array before calling `length` on it to avoid potential runtime errors.
```diff
- total: followed?.length
+ total: Array.isArray(followed) ? followed.length : 0
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->